.goreleaser.yml

```diff
@@ -17,9 +22,57 @@ builds:
     env:
       - CGO_ENABLED=0
     ldflags:
-      - -s -w -X main.version={{if .IsSnapshot}}dev{{else}}{{.Version}}{{end}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}
+      - -s -w -X main.version={{if .IsSnapshot}}{{.Env.SNAPSHOT_VERSION}}{{else}}{{.Version}}{{end}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}
     binary: influx
 
-# Do not make github release
-release:
-  disable: true
+# TODO: Do we want to publish deb/rpms for the CLI before we release 2.1.0 of the server?
```

Comment:
Has Product weighed in on this yet? My feeling is that if we are developing and building the CLI separately, we should make packages available, but also this would require some nontrivial documentation of what the user path is and what we are prepared to support in terms of conflicts.

Reply:
Not yet, will ping Russ

Reply:
Product aside, I was worried about releasing a 2.1 deb/rpm into the wild when the new package contains a file that's already present in `influxdb2` (`/usr/bin/influx`). It looks like the deb system has a way of specifying the breakage in this case: https://www.debian.org/doc/debian-policy/ch-relationships.html#s-replaces. Need to find if rpm has a similar concept

Reply:
RPM doesn't have a 1:1 concept. What I found is:
* `yum install` doesn't work when there's a shared file between the two packages
* `rpm -i --replacefiles` does work as long as you install the CLI after the server

The work-around should only be needed until we release a version of `influxdb2` that only contains the server.

Reply:
Interesting. yeah it does seem a bit messy, I wonder if we could release it initially as a series of rc releases to make it clear that it's not yet "GA" until we get the conflict worked out

Reply:
For now I've added a "conflicts" clause to the RPMs. That'll prevent anyone from installing `influx-cli` if they already have `influxdb2`, which is annoying but hopefully not as annoying as installing `influx-cli` and discovering that the `influxd` binary has disappeared.